scene/main/viewport.cpp

```diff
@@ -5332,6 +5350,17 @@ Viewport::Viewport() {
 
 	viewport = RenderingServer::get_singleton()->viewport_create();
 	texture_rid = RenderingServer::get_singleton()->viewport_get_texture(viewport);
+	/*{
```

Comment:
This needs some comment clarifying why this is commented out and what needs to be done to fix the code, or simply remove it